docsrc/contributors/fx_converters.rst

```diff
@@ -0,0 +1,211 @@
+.. _dynamo_conversion:
+
+Dynamo Converters
+==================
+The dynamo converter library in Torch-TensorRT is located in ``TensorRT/py/torch_tensorrt/dynamo/conversion``.
+
+
+
+Steps
+==================
+
+Operation Set
+-------------------
+The converters in dynamo are produced by ``aten_trace`` and falls under ``aten_ops_converters`` ( FX earlier had ``acc_ops_converters``, ``aten_ops_converters`` or ``nn_ops_converters`` depending on the trace through which it was produced).  The converters are registered using  ``dynamo_tensorrt_converter`` for dynamo. The function decorated
+has the arguments - ``network, target, args, kwargs, name``,  which is common across all the operators schema.
+These functions are mapped in the ``aten`` converter registry dictionary (at present a compilation of FX and dynamo converters, FX will be deprecated soon), with key as the function target name.
+    
+    * aten_trace is produced by ``torch_tensorrt.dynamo.backend.compile`` or  ``torch_tensorrt.dynamo.backend.export``.
```

Comment:
This isnt the case anymore right? its torch_tensorrt.dynamo.compile and torch.compile(backend="torch_tensorrt")

Reply:
@narendasan @gs-olive . Changed this. Could you verify once?